src/SFA.DAS.EAS.Web/Views/Transfers/TransferDetails.cshtml

```diff
@@ -68,36 +68,66 @@
                 <p>@item.SummaryDescription</p>
             }
         </div>
-        <hr class="hr-bottom" />
+        @if (Model.Data.PendingApproval)
+        {
+            <hr class="hr-bottom"/>
+        }
     </div>
+    @if (!Model.Data.PendingApproval)
+    {
+        <div class="column-full">
+            <hr class="hr-top"/>
+            <div class="column-one-third">
+                @Model.Data.TransferApprovalStatus By
+            </div>
+            <div class="column-two-thirds">
+                @Model.Data.TransferApprovalSetBy
+            </div>
+        </div>
+        <div class="column-full">
+            <hr class="hr-top" />
+            <div class="column-one-third">
+                @Model.Data.TransferApprovalStatus On
+            </div>
+            <div class="column-two-thirds">
+                @($"{Model.Data.TransferApprovalSetOn:dd MMM yyyy}")
+            </div>
+        </div>
+        <div class="column-full">
+            <hr class="hr-bottom" />
+        </div>
+    }
 </div>
 
-<div class="grid-row">
-    <div class="column-two-thirds">
-        <form method="POST" action="@Url.Action("TransferApproval")" onsubmit="sfa.tagHelper.submitRadioForm('@ViewBag.Title')">
+@if (Model.Data.PendingApproval)
+{
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <form method="POST" action="@Url.Action("TransferApproval")" onsubmit="sfa.tagHelper.submitRadioForm('@ViewBag.Title')">
 
-            <fieldset role="radiogroup" aria-required="true" class="approve-transfer">
-                <div class="form-error-group form-group @Html.AddClassIfPropertyInError("ApprovalConfirmed", "error")">
-                    <h2 class="heading-medium" for="">Do you want to approve this transfer request?</h2>
-                    <div>
-                        @Html.ValidationMessage("ApprovalConfirmed", "Please choose an option", new { id = "error-message-ApprovalConfirmed" })
+                <fieldset role="radiogroup" aria-required="true" class="approve-transfer">
+                    <div class="form-error-group form-group @Html.AddClassIfPropertyInError("ApprovalConfirmed", "error")">
+                        <h2 class="heading-medium" for="">Do you want to approve this transfer request?</h2>
+                        <div>
+                            @Html.ValidationMessage("ApprovalConfirmed", "Please choose an option", new {id = "error-message-ApprovalConfirmed"})
+                        </div>
+                        <label class="block-label selection-button-radio" for="ApprovalConfirmed-True">
+                            @Html.RadioButton("ApprovalConfirmed", "True", new {id = "ApprovalConfirmed-True", dataOptionName = "Yes", onclick = $"sfa.tagHelper.radioButtonClick('{ViewBag.Title}', 'Yes')"})
+                            Yes, use approve the request
```

Comment:
Not introduced in this PR, but shouldn't his this read 'Yes, approve the request', rather than 'Yes, use approve the request'?

Reply:
Done